app/modules/intelligence/agents/chat_agents/multi_agent/utils/multimodal_utils.py

```
@@ -69,8 +117,80 @@ def validate_and_build_data_url(image_data: dict, attachment_id: str) -> Optiona
+            logger.warning(
+                "Document URL for %s is not HTTPS (%s); skipping DocumentUrl part",
+                attachment_id,
+                document_url[:64],
+            )
+            return None
+
+        return document_url
+
+    except Exception as e:
+        logger.error(
+            f"Failed to validate document {attachment_id}: {str(e)}",
```

Ruff (0.15.9)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)